<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>173</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Qt Mac Toolbar Example</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <property name="fieldGrowthPolicy">
    <enum>QFormLayout::FieldsStayAtSizeHint</enum>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="displayModeLabel">
     <property name="text">
      <string>Display mode:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="displayModeComboBox">
     <item>
      <property name="text">
       <string>IconOnly</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TextOnly</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TextBesideIcon</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TextUnderIcon</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>FollowStyle</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="sizeModeLabel">
     <property name="text">
      <string>Size mode:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="sizeModeComboBox">
     <item>
      <property name="text">
       <string>Default</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Regular</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Small</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="visibleCheckBox">
     <property name="text">
      <string>Visible</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="showsBaselineSeparatorCheckBox">
     <property name="text">
      <string>Shows baseline separator</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QCheckBox" name="allowsUserCustomizationCheckBox">
     <property name="text">
      <string>Allows user customization</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="showCustomizationSheetPushButton">
     <property name="text">
      <string>Show customization sheet</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>displayModeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Window</receiver>
   <slot>changeDisplayMode(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>337</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeModeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Window</receiver>
   <slot>changeSizeMode(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>changeDisplayMode(int)</slot>
  <slot>changeSizeMode(int)</slot>
 </slots>
</ui>
